library/include/ck/library/tensor_operation_instance/gpu/grouped_gemm_fixed_nk.hpp

```diff
@@ -180,6 +207,22 @@ struct DeviceOperationInstanceFactory<
             }
         }
 
+        // bf16_i8_input
+        if constexpr(is_same_v<ADataType, bhalf_t> && is_same_v<BDataType, int8_t> &&
```

Comment:
Please add conditional compilation if `CK_ENABLE_BF16` and `CK_ENABLE_I8` symbols are defined.